src/PointwiseFunctions/Punctures/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Distributed under the MIT License.
```

Comment:
This whole `AdmMass` thing sounds very `GeneralRelativity`. Why not put it in the Gr directory? If I were looking for this that's where I would look, not `Punctures`

Reply:
This form of the ADM mass is specific to the puncture system. The XCTS system has another form (split into a volume integral and an excision surface integral) and in GH there's yet another form.